cloudbuild.yaml

```diff
@@ -312,18 +312,10 @@ steps:
         for version in "${!versionsAndRegions[@]}"
         do
           region=${versionsAndRegions[$version]}
-          if [ $cloudProduct = generic ]
-          then
-            featureWithGate="CustomFasSyncInterval=false&SDKGracefulTermination=false&StateAllocationFilter=false&PlayerAllocationFilter=true&PlayerTracking=true&ResetMetricsOnDelete=false&PodHostname=false&SplitControllerAndExtensions=false&FleetAllocationOverflow=true&Example=true"
-            featureWithoutGate=""
-            testClusterLocation="${region}-c"
```

Comment:
We cannot do this before move the Standard cluster to regional.

Reply:
Ooooooh I see! Okay I completely missed that. Thanks!

Reply:
I think it's just the `featureWithGate` and `featureWithoutGate` variables that get collapsed.

Reply:
I see, so that means there isn't a need for a `featureWithoutGate` anymore. Just one variable `featurewithGate`

Reply:
Correct. You should be able to clean up the usages of `featureWithoutGate` and (unless I'm misreading it) collapse the two places where `featurewithGate` is set into one.